lib/mysqlDao.js

```diff
@@ -193,31 +193,47 @@ class MysqlDao {
 
     /**
      * @param {object} whereClause
+     * @param {object} options
```

Comment:
We should change the options argument to be as follows:

```
{
  sortBy: {
    col1: 'asc',
    col2: 'desc'
  },
  limit: {int},
  offset: {int} 
}
```

The order of the sortBy properties determine the sort order.  High level is that we have to be able to add multiple columns in a particular order with asc/desc for each.

Next, Factor out a common JSDoc comment so you can simply use a name for it.  Like:

```
/**
 * @name {MysqlDaoOptions}
 * @type {Object}
 * @property {Object} [sortBy] - properties should be column names, values are 'asc'/'desc'
 * @property {int} [limit]
 * @property {int} [offset] - you have to provide a limit if you provide an offset
 */
```

You can use that as the type instead of duplicating the options def for every function